documentation/asciidoc/computers/config_txt/conditional.adoc

```diff
@@ -123,19 +123,96 @@ cmdline=cmdline-recovery.txt
 
 The raw value of the `PM_RSTS` register at bootup is available via `/proc/device-tree/chosen/bootloader/rsts` and the final partition number used for booting is available via `/proc/device-tree/chosen/bootloader/partition`. These are big-endian binary values.
 
-=== The `[boot_partition=N]` filter
-The `boot_partition` filter can be used to select alternate OS files (e.g. `cmdline.txt`) to be loaded, depending on which partition `config.txt` was loaded from after processing `autoboot.txt`. This is intended for use with an `A/B` boot-system with `autoboot.txt` where it is desirable to be able to have identical files installed to the boot partition for both the `A` and `B` images.
+=== The expression filter
 
-Example `config.txt` - select the matching root filesystem for the `A/B` boot file-system.
+The expression filter provides support for comparing unsigned integer `boot variables` with constants using a simple set of expressions. It is intended to support OTA update mechanisms, debug and test.
+
+* The `boot variables` are `boot_count`, `boot_arg1`, `boot_partition` and `partition`. 
+* Boot variables in the expressions case must always be lower case.
+* Constants may be written as decimal or as hex.
```

Comment:
"Constants (i.e `MASK` and `VALUE` in the list below) may be..." ?

Reply:
Yes, but I think that's obvious

Reply:
The syntax already states that MASK and VALUE are constants. I've change the text to say Numerical constants may be ...